help/sources/tutorials/ui/create/data-partners/acxiom-prospecting-data-import.md

```diff
@@ -0,0 +1,154 @@
+
+| Credential | Description |
+| --- | --- |
+| Acxiom authentication key | The authentication key. You can retrieve this value from the [!DNL Acxiom] team. |
+| S3 Access key | The access key ID for your bucket. You can retrieve this value from the [!DNL Acxiom] team. |
+| S3 Secret key | The secret key ID for your bucket. You can retrieve this value from the [!DNL Acxiom] team. |
+| Bucket name | This is your bucket where files will be shared. You can retrieve this value from the [!DNL Acxiom] team. |
+
+## Connect your [!DNL Acxiom] Account
+
+Source Configuration and Authentication - Defined S3 account associated with [!DNL Acxiom] prospect response.
+
+* **Existing Account** - Accounts already defined using the Prospecting Data for Adobe AEP card will appear here for reuse.  These will appear in a list below and when selected provides details on the account.
+  <br>![The sources catalog with the Acxiom source selected.](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-existing-account.png)
+* **New Account** - Define a new [!DNL Acxiom] Managed S3 location
+  * **Account Name (Required)** - The name the account will be saved under 
+  * **Description** - Short explanation of the account purpose
+  * **Account Authentication** 
+    * **Enter [!DNL Acxiom] authentication key (Required)** - [!DNL Acxiom] provided value used for account approval.  This will be a must match the proper value before connection to the database can be made.  The value must be 24 characters and can only include characters A-Z, a-z, 0-9.
+    * **S3 access key (Required)** - Reference to Amazon S3 location, provided by admin when S3 role permissions are defined.
+    * **S3 secret key (Required)** - Reference to Amazon S3 location, provided by admin when S3 role permissions are defined.
+    * **s3SessionToken** - Optional authentication token value used when connecting to S3
+    * **serviceUrl** - Optional URL location to be used when connecting to S3 in non-standard location
+    * **Bucket name** - Optional name of the S3 bucket set up on S3 as a starting path in data selection
+    * **Folder path** - If subdirectories in a bucket are used a path could be provided as a starting path in data selection
+    * **Connect to Source** - This button will only be enabled once an Account Name, a valid [!DNL Acxiom] authentication key, and a properly formatted S3 access key and session key is provided.
+
+
+## Data Selection ##
+selecting file in the desired bucket and sub directory, can be performed here.  A preview of the data can be provided once delimiter and compression type is defined.  While JSON and Parquet are listed, we have seen no indication that these formats should be expected by [!DNL Acxiom] processing and will not be supported in the long term.
+<br>    ![File preview.](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-preview.png)
+
+## Dataset Definition  ##
```

Comment:
- Remove extra ##
- Update to "Provide dataset and dataflow details"

Reply:
Cleaned up occurrences of extra ## and changed title of section to suggestion